presto-main/src/test/java/com/facebook/presto/operator/aggregation/AbstractTestApproximateCountDistinct.java

```diff
@@ -108,14 +114,14 @@ public void testMultiplePositions(double maxStandardError)
         }
 
         assertLessThan(stats.getMean(), 1.0e-2);
-        assertLessThan(Math.abs(stats.getStandardDeviation() - maxStandardError), 1.0e-2);
+        assertLessThan(Math.abs(stats.getStandardDeviation()), 1.0e-2 + maxStandardError);
```

Comment:
static import `abs`

Reply:
It looks as a separate bug fix. Try to keep the independent changes in separate commits.